content/nim/monitoring/security-monitoring/configure/set-up-app-protect-instances.md

```diff
@@ -1,17 +1,15 @@
 ---
 title: Create App Protect WAF instances for Security Monitoring
-description: Learn how to set up F5 NGINX App Protect data plane instances for use with
-  the NGINX Security Monitoring and NGINX Instance Manager.
-toc: true
 weight: 100
+toc: true
 type: how-to
 product: NIM
 docs: DOCS-1107
 ---
 
 ## Overview
 
-F5 NGINX Security Monitoring supports the following use cases:
+The Security Monitoring module supports the following use cases:
```

Comment:
I'd avoid calling Sec Mon a module -- it'll be a feature of NIM 2.19 (automatically installed w/ NIM).